src/ModuleTestLite.cs

```diff
@@ -341,7 +341,12 @@ private void triggerFailure(int type)
 					type = (int)failureTypes.PERFLOSS;
 				}
 			}
-			failure_du += failureData[type];
+            // No failure data until the craft has launched
+            if (runTime > clampTime)
```

Comment:
IMO the user should get at least _some_ du for an ignition failure on the pad, and perhaps also for a failure during spool-up.  At least, for the first ignition; if they sit on the pad spamming Z and X they shouldn't be able to get arbitrary amounts of data that way.

So I think we should store a "has this engine ever ignited" bit, and if it's false then allow ignition-failure du.  But I'm not entirely sure how to deal with spool-up failures.  (The alternative is just to tell players that abusing pad-failure du is cheating; if they want to cheat, they're only cheating themselves.)